### PR TITLE
Dynamic  content and Card Popup Implemented

### PR DESCRIPTION
**Pull request by ometman**
Created a card-popup branch
Created a javascript file, card-popup.js

Inside the script I have created:

- An object storing all the various card information
- A dynamic content for the works section for both mobile and desktop
- A popup modal that shows details of the user-selected card based on figma design
- A function the handles mobile and desktop popups
- Links to the live demo and code source.

Index.html and css files were modified to fit popups
Html, css and js linters were  run and checked locally and passed

### CODE REVIEW

File: css/styles1440.css [hunk 1]
```diff
@@ -339,11 +339,100 @@
...
+    width: 100%;
+    height: 300vh;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    padding: 0;
+    z-index: 555;
+    background: rgba(194, 200, 209, 0.85);
+  }
```
Comment (Qoosim) [inline css/styles1440.css:353]: - [OPTIONAL] Wonderful job on making your popup window. However, I noticed the popup does not fix while trying to scroll downward as oppose to the best practice. I would recommend you to make it fixed once it has popped up until a new action is performed to make it disappear. 

[popup-desktop.webm](https://user-images.githubusercontent.com/26597432/218136160-d0455bba-b21a-4bd1-a43f-132a1febcd9d.webm)

**Review by Qoosim — CHANGES_REQUESTED** [review]
Hi @team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
-   No linter errors. :heavy_check_mark: 
-   Good README file. :heavy_check_mark: 
-   Correct GitHub Flow. :heavy_check_mark: 



## Required Changes ♻️

- [ ] The link to the license on your repo is broken. Kindly follow this [link](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to properly set it up for your project.

license


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/styles1440.css (hunk 1 above)
Comment (ometman) [reply css/styles1440.css:353]: Thank you very for bringing these to my attention. The video was good.

**Review by anagudelogu — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 🚧 

Hi @ometman 👋🏼 
I'm Andres.

You've done a great job implementing some of the requested changes for this milestone 🚀, but there are still some which haven't been addressed yet. You're getting closer, keep up the good work! 🤖 🎆 



 ## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 


 ## Required Changes ♻️
_Check the comments under the review._

- [ ] You did a good job working on your project documentation! 💯 However, as requested by the previous reviewer, please ensure that you add a LICENSE and include a link in your README file. Microverse suggests that you use the MIT one, you could set it following the steps from [GitHub's official docs](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). 🤓 🔧 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ometman** [comment]
Thank you. But I added the license link in the card-popup branch, so that when reviewed and merged . I can update the main.

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @ometman, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ometman** [comment]
Thank you. I appreciate your good  review work.

**Pull request merged**